test/NuGet.Configuration.Test/PackageSourceProviderTests.cs

```diff
@@ -798,19 +796,17 @@ public void SavePackageSourcesWithOnlyClear()
 
                 // Assert
                 Assert.Equal(
-                   TestFilesystemUtility.FormatXmlString(
                        @"<?xml version=""1.0"" encoding=""utf-8""?>
 <configuration>
     <packageSources>
-        <clear /> 
+        <clear />
         <add key=""test3"" value=""https://test3.net"" />
     </packageSources>
     <disabledPackageSources>
-        <clear />
-    </disabledPackageSources>
+        <clear /></disabledPackageSources>
```

Comment:
It ends up looking messed up in this scenario (the `clear` element is the last element) - I guess we can special case for this if it's pertient?